ui/admin/app/components/tools/ToolCatalog.tsx

```diff
@@ -116,6 +120,49 @@ export function ToolCatalog({
             </CommandList>
         </Command>
     );
+
+    function filterToolCatalogBySearch(
+        toolCategories: [string, ToolCategory][]
+    ) {
+        return toolCategories.reduce(
```

Comment:
something I want to start doing Purely for readability is adding the output type annotation to the reduce function instead of just casting the `initialValue`

```suggestion
        return toolCategories.reduce<[string, ToolCategory][]>(
```